src/bmark/run_solver.jl

```diff
@@ -3,9 +3,11 @@ export display_header, solve_problems, solve_problem
 type SkipException <: Exception
 end
 
+const uncstats = [:obj, :dual_feas, :neval_obj, :neval_grad, :neval_hprod, :iter, :elapsed_time, :status]
```

Comment:
Why would we separate stats for unconstrained problems? I would have put them all together. Measures such as `:neval_cons` will remain zero for unconstrained problems.